en/flight_modes/offboard.md

```diff
@@ -0,0 +1,42 @@
+# Offboard Flight Mode
+
+The vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi). The setpoint can be provided by APIs like [DroneCore](http://dronecore.io/) or [MAVROS](https://github.com/mavlink/mavros).
+
+> **Note** 
+>  * This mode requires position information (GPS or optical flow).
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+>  * The vehicle must be armed before this mode can be engaged.
+
+## Description
+
+Offboard mode is primarily used for controlling vehicle movement and attitude, and supports only a very limited set of MAVLink commands (more may be supported in future). It can be used to:
+* Control vehicle position or velocity (`SET_POSITION_TARGET_LOCAL_NED`). 
+* Control vehicle attitude/orientation (`SET_ATTITUDE_TARGET`)
+
+> **Note** Velocity and attitude commands must be resent regularly at a rate of around (TBD Hz) or PX4 will stop the vehicle. 
+
+Other operations, like taking off, land, return to launch, arming, are best handled using the appropriate modes. Operations like uploading, downloading missions can be performed in any mode.
+
+Offboard mode requires an active connection to a remote system. If the connection is lost the vehicle will attempt to land or perform some other failsafe action (after a timeout ([COM_OF_LOSS_T](#COM_OF_LOSS_T)). The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL_RC_ACT](#COM_OBL_RC_ACT).
+
+
+## Offboard Parameters
+
+*Offboard mode* is affected by the following parameters:
+
+Parameter | Description
+--- | ---
+<span id="COM_OF_LOSS_T"></span>[COM_OF_LOSS_T](../advanced_config/parameter_reference.md#COM_OF_LOSS_T) | Time-out (in seconds) to wait when offboard connection is lost before triggering offboard lost failsafe (`COM_OBL_ACT` and `COM_OBL_RC_ACT`)
+<span id="COM_OBL_ACT"></span>[COM_OBL_ACT](../advanced_config/parameter_reference.md#COM_OBL_ACT) | Mode to switch to if offboard control is lost when *not* connected to RC (Values are - 0: [Land](../flight_modes/land.md), 1: [Hold](../flight_modes/hold.md), 2: [Return ](../flight_modes/rtl.md)).
+<span id="COM_OBL_RC_ACT"></span>[COM_OBL_RC_ACT](../advanced_config/parameter_reference.md#COM_OBL_RC_ACT) | Mode to switch to if offboard control is lost while still connected to RC control (Values are - 0: *Position*, 1: [Altitude](../flight_modes/altitude.md), 2: *Manual*, 3: [Return ](../flight_modes/rtl.md), 4: [Land](../flight_modes/land.md)).

```

Comment:
I would need to look at the code and play with this to confirm. I'll let you know once I get the chance.

Reply:
No urgency. These are the important ones I can see that most users (ie not developers) would actually want to play with.